### ISSUE TITLE
Will the traffic be recognized by the firewall? What encryption protocols and algorithms are used?

### ISSUE THREAD

**Issue by ghost** [pbxforce/vpn-server-openvpn#4]
Will the traffic be recognized by the firewall? What encryption protocols and algorithms are used?

**Comment by pbxforce** [comment]
Yes. Traffic will be recognized by the firewall and must be allowed by opening a certain port over TCP/UDP (though this process will be automated with the script). VPN tunnel traffic is encrypted with symmetric key using AES-256-CBC algorithm and RSA algorithm is used for key-exchange process.

**Comment by pbxforce** [comment]
> Will the traffic be recognized by the firewall? What encryption protocols and algorithms are used?

Yes. Traffic will be recognized by the firewall and must be allowed by opening a certain port over TCP/UDP (though this process will be automated with the script). VPN tunnel traffic is encrypted with symmetric key using AES-256-CBC algorithm and RSA algorithm is used for key-exchange process.

**Comment by ghost** [comment]
After I used this script, the server IP was blocked and the location was in China.

**Comment by pbxforce** [comment]
> After I used this script, the server IP was blocked and the location was in China.

It doesn't make sense. IP location depends on where your server is located. Script does not perform any operation on IP manipulation. IP address must be the public address of the server where the script is being executed.

**Comment by pbxforce** [comment]
> After I used this script, the server IP was blocked and the location was in China.

I just tested on AWS EC2 instance, everything working fine.

**Comment by ghost** [comment]
I'm referring to the IP being blocked, the VPN detected.

**Comment by navidlukas** [comment]
> I'm referring to the IP being blocked, the VPN detected.

OpenVPN does not have stealth/obfuscation...the application signature can be detected by firewalls. Forget this with China and Iran.
